### ISSUE TITLE
deprecations of dependencies

### ISSUE THREAD

**Issue by geoHeil** [OSGeo/homebrew-osgeo4mac#309]
homebrew notifies me of:
```
Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/gdal2-python.rb:66:in `<class:Gdal2Python>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :python3' is deprecated!
Use 'depends_on "python3"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/gdal2-python.rb:67:in `<class:Gdal2Python>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/pyspatialite.rb:29:in `<class:Pyspatialite>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :gpg' is deprecated!
Use 'depends_on "gnupg"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/qca-qt4.rb:43:in `<class:QcaQt4>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :postgresql' is deprecated!
Use 'depends_on "postgresql"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/grass7.rb:50:in `<class:Grass7>'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling 'depends_on :mysql' is deprecated!
Use 'depends_on "mysql"' instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/grass7.rb:51:in `<class:Grass7>'
Please report this to the osgeo/osgeo4mac tap!
```

**Comment by dd-a** [comment]
same here !

**Comment by luisspuerto** [comment]
same here #306

**Comment by kagou** [comment]
I confirm this bug too. Can't build Qgis 3 (dev)

**Comment by nickrobison** [comment]
Thanks for reporting. I believe this has been resolved in #308.
